addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -455,11 +459,6 @@ var FileWidget = SearchWidget.extend({
                 self.$media.css(style);
             }
 
-            if (self.options.onUpload) {
-                // We consider that when selecting an image it is as if we upload it in the html content.
-                self.options.onUpload([img]);
```

Comment:
This was added by CHM with https://github.com/odoo/odoo/commit/b26ad79a8c3f02726ed3f22f73faf49a2a272d1d and handled something useful... worth checking if it is still there with the new editor and telling CHM to remember to adapt its own work when it changes its own work :/ When you remove stuff that seems useless, try always checking it is not a broken feature first ;)

Reply:
When introduced it was used only in `addons/web_editor/static/src/js/editor/rte.summernote.js` which was later removed in f296992317e96562c66bd7ad59a5080d6c551ed5.

Reply:
CHM's answer about this: "/widgets/ will be removed" ...